openwisp-monitoring/files/lib/openwisp-monitoring/interfaces.lua

```diff
@@ -128,8 +128,9 @@ function interfaces.get_addresses(name)
           if utils.starts_with(addr, 'fe80') then
             proto = 'static'
           else
-            local ula = uci_cursor.get('network', 'globals', 'ula_prefix')
-            local ula_prefix = utils.split(ula, '::')[1]
+            local ula_prefix = uci_cursor.get('network', 'globals', 'ula_prefix') or
+                                 ''
```

Comment:
Is the formatter doing this? It looks really ugly, doesn't it.
Can we find a way to avoid this weirdness and keep it on the line above?